src/pages/Home/index.tsx

```diff
@@ -1,11 +1,16 @@
 import { Container } from 'react-bootstrap';
 import BobaContainer from '../../components/BobaContainer';
-
-const Homepage = () => {
+import { BobaProps } from '../../assets/sampleBobaAPI';
+interface HomePageProps {
```

Comment:
Add new line. Did you run eslint?

Reply:
even i run yarn , it is not moving or add extra lines
i just tested it.